### ISSUE TITLE
Got a very small ACCEL and high MPJPE after i did some modification to the model.

### ISSUE THREAD

**Issue by sunwenzhang1996** [mkocabas/VIBE#178]
hi, appreciate your work, i did some modification to the regressor layers and retrained the model. and get some strange evaluation result. the training log is like this, i got extreme low ACCEL and other values are too high. could you help me with this probelm?

 'DATASET': CfgNode({'SEQLEN': 8, 'OVERLAP': 0.5}),
 'DEBUG': False,
 'DEBUG_FREQ': 5,
 'DEVICE': 'cuda',
 'EXP_NAME': 'vibe',
 'LOGDIR': 'results/vibe_tests/21-02-2021_16-09-53_vibe',
 'LOSS': {'D_MOTION_LOSS_W': 0.5,
          'KP_2D_W': 300.0,
          'KP_3D_W': 300.0,
          'POSE_W': 60.0,
          'SHAPE_W': 0.06},
 'MODEL': {'TEMPORAL_TYPE': 'gru',
           'TGRU': {'ADD_LINEAR': True,
                    'BIDIRECTIONAL': False,
                    'HIDDEN_SIZE': 1024,
                    'NUM_LAYERS': 2,
                    'RESIDUAL': True}},
 'NUM_WORKERS': 8,
 'OUTPUT_DIR': 'results/vibe_tests',
 'SEED_VALUE': -1,
 'TRAIN': {'BATCH_SIZE': 16,
           'DATASETS_2D': ['Insta'],
           'DATASETS_3D': ['ThreeDPW', 'MPII3D'],
           'DATASET_EVAL': 'ThreeDPW',
           'DATA_2D_RATIO': 0.6,
           'END_EPOCH': 30,
           'GEN_LR': 5e-05,
           'GEN_MOMENTUM': 0.9,
           'GEN_OPTIM': 'Adam',
           'GEN_WD': 0.0,
           'LR_PATIENCE': 5,
           'MOT_DISCR': {'ATT': {'DROPOUT': 0.2,
                                 'LAYERS': 3,
                                 'SIZE': 1024},
                         'FEATURE_POOL': 'attention',
                         'HIDDEN_SIZE': 1024,
                         'LR': 0.0001,
                         'MOMENTUM': 0.9,
                         'NUM_LAYERS': 2,
                         'OPTIM': 'Adam',
                         'UPDATE_STEPS': 1,
                         'WD': 0.0001},
           'NUM_ITERS_PER_EPOCH': 500,
           'PRETRAINED': '',
           'PRETRAINED_REGRESSOR': 'data/vibe_data/spin_model_checkpoint.pth.tar',
           'RESUME': '',
           'START_EPOCH': 0}}
2021-02-21 16:13:28,306 (1000/1000) | Total: 0:03:31 | ETA: 0:00:01 | loss: 14.4873 | loss_2d_kp: 0.65 | loss_3d_kp: 2.80 | loss_shape: 0.55 | loss_pose: 4.42 | data: 0.00 | forward: 0.02 | loss: 0.00 | backward: 0.03 | batch: 0.05
2021-02-21 16:13:31,850 (80/80) | batch: 34.02ms | Total: 0:00:03 | ETA: 0:00:01
2021-02-21 16:13:35,503 Epoch 0, MPJPE: 261.6414, PA-MPJPE: 199.7043, ACCEL: 2.0116, PVE: 307.3648, ACCEL_ERR: 8.9401,
2021-02-21 16:13:35,504 Epoch 1 performance: 199.7043
2021-02-21 16:13:37,341 Best performance achived, saving it!
2021-02-21 16:16:54,585 (1000/1000) | Total: 0:03:15 | ETA: 0:00:01 | loss: 6.2192 | loss_2d_kp: 0.41 | loss_3d_kp: 2.49 | loss_shape: 0.39 | loss_pose: 2.22 | data: 0.22 | forward: 0.02 | loss: 0.00 | backward: 0.03 | batch: 0.26
2021-02-21 16:16:57,939 (80/80) | batch: 33.18ms | Total: 0:00:03 | ETA: 0:00:01
2021-02-21 16:17:01,633 Epoch 1, MPJPE: 254.8530, PA-MPJPE: 183.8711, ACCEL: 2.4942, PVE: 301.6526, ACCEL_ERR: 9.1273,
2021-02-21 16:17:01,634 Epoch 2 performance: 183.8711
2021-02-21 16:17:04,253 Best performance achived, saving it!
2021-02-21 16:19:50,759 (1000/1000) | Total: 0:02:43 | ETA: 0:00:01 | loss: 5.9241 | loss_2d_kp: 0.57 | loss_3d_kp: 2.79 | loss_shape: 0.42 | loss_pose: 3.49 | data: 0.00 | forward: 0.02 | loss: 0.00 | backward: 0.03 | batch: 0.05
2021-02-21 16:19:53,946 (80/80) | batch: 31.3ms | Total: 0:00:03 | ETA: 0:00:01
2021-02-21 16:19:57,591 Epoch 2, MPJPE: 252.6574, PA-MPJPE: 183.9710, ACCEL: 2.4984, PVE: 298.0957, ACCEL_ERR: 9.0420,
2021-02-21 16:19:57,592 Epoch 3 performance: 183.9710
2021-02-21 16:22:19,262 (1000/1000) | Total: 0:02:19 | ETA: 0:00:01 | loss: 5.7484 | loss_2d_kp: 0.43 | loss_3d_kp: 2.52 | loss_shape: 1.13 | loss_pose: 2.84 | data: 0.00 | forward: 0.02 | loss: 0.01 | backward: 0.03 | batch: 0.05
2021-02-21 16:22:22,467 (80/80) | batch: 31.09ms | Total: 0:00:03 | ETA: 0:00:01
2021-02-21 16:22:26,091 Epoch 3, MPJPE: 246.4723, PA-MPJPE: 172.6609, ACCEL: 2.6620, PVE: 287.8747, ACCEL_ERR: 9.0917,
2021-02-21 16:22:26,092 Epoch 4 performance: 172.6609
2021-02-21 16:22:28,105 Best performance achived, saving it!
2021-02-21 16:24:44,866 (1000/1000) | Total: 0:02:14 | ETA: 0:00:01 | loss: 5.6328 | loss_2d_kp: 0.62 | loss_3d_kp: 3.08 | loss_shape: 0.14 | loss_pose: 2.84 | data: 0.00 | forward: 0.02 | loss: 0.00 | backward: 0.03 | batch: 0.05
2021-02-21 16:24:48,123 (80/80) | batch: 31.52ms | Total: 0:00:03 | ETA: 0:00:01
2021-02-21 16:24:51,926 Epoch 4, MPJPE: 245.7085, PA-MPJPE: 174.9934, ACCEL: 2.7203, PVE: 286.2461, ACCEL_ERR: 9.0614,
2021-02-21 16:24:51,927 Epoch 5 performance: 174.9934
2021-02-21 16:27:00,857 (1000/1000) | Total: 0:02:06 | ETA: 0:00:01 | loss: 5.5802 | loss_2d_kp: 0.43 | loss_3d_kp: 1.91 | loss_shape: 0.24 | loss_pose: 1.94 | data: 0.00 | forward: 0.02 | loss: 0.00 | backward: 0.03 | batch: 0.05
2021-02-21 16:27:04,074 (80/80) | batch: 31.34ms | Total: 0:00:03 | ETA: 0:00:01
2021-02-21 16:27:07,701 Epoch 5, MPJPE: 242.2675, PA-MPJPE: 168.9271, ACCEL: 2.8045, PVE: 283.9822, ACCEL_ERR: 9.1182,
2021-02-21 16:27:07,702 Epoch 6 performance: 168.9271
2021-02-21 16:27:09,866 Best performance achived, saving it!
2021-02-21 16:29:17,828 (1000/1000) | Total: 0:02:05 | ETA: 0:00:01 | loss: 5.5266 | loss_2d_kp: 1.30 | loss_3d_kp: 1.83 | loss_shape: 0.36 | loss_pose: 2.16 | data: 0.00 | forward: 0.02 | loss: 0.01 | backward: 0.03 | batch: 0.05
2021-02-21 16:29:21,121 (80/80) | batch: 32.25ms | Total: 0:00:03 | ETA: 0:00:01
2021-02-21 16:29:24,799 Epoch 6, MPJPE: 237.7331, PA-MPJPE: 160.2946, ACCEL: 2.9143, PVE: 277.6658, ACCEL_ERR: 9.1748,
2021-02-21 16:29:24,800 Epoch 7 performance: 160.2946
2021-02-21 16:29:26,828 Best performance achived, saving it!
2021-02-21 16:31:32,193 (1000/1000) | Total: 0:02:03 | ETA: 0:00:01 | loss: 5.3415 | loss_2d_kp: 0.44 | loss_3d_kp: 1.75 | loss_shape: 0.15 | loss_pose: 2.44 | data: 0.32 | forward: 0.02 | loss: 0.00 | backward: 0.03 | batch: 0.36
2021-02-21 16:31:35,209 (80/80) | batch: 29.72ms | Total: 0:00:02 | ETA: 0:00:01
2021-02-21 16:31:38,883 Epoch 7, MPJPE: 232.6970, PA-MPJPE: 147.0250, ACCEL: 3.0895, PVE: 270.2666, ACCEL_ERR: 9.2731,
2021-02-21 16:31:38,884 Epoch 8 performance: 147.0250
2021-02-21 16:31:41,379 Best performance achived, saving it!
2021-02-21 16:33:45,499 (1000/1000) | Total: 0:02:01 | ETA: 0:00:01 | loss: 5.1507 | loss_2d_kp: 0.43 | loss_3d_kp: 1.82 | loss_shape: 0.24 | loss_pose: 1.76 | data: 0.35 | forward: 0.02 | loss: 0.00 | backward: 0.03 | batch: 0.40
2021-02-21 16:33:48,817 (80/80) | batch: 32.12ms | Total: 0:00:03 | ETA: 0:00:01
2021-02-21 16:33:52,525 Epoch 8, MPJPE: 230.4150, PA-MPJPE: 144.0851, ACCEL: 3.0268, PVE: 268.7804, ACCEL_ERR: 9.2585,
2021-02-21 16:33:52,525 Epoch 9 performance: 144.0851
2021-02-21 16:33:55,046 Best performance achived, saving it!
2021-02-21 16:35:56,493 (1000/1000) | Total: 0:01:58 | ETA: 0:00:01 | loss: 5.0513 | loss_2d_kp: 0.45 | loss_3d_kp: 1.60 | loss_shape: 0.28 | loss_pose: 2.03 | data: 0.00 | forward: 0.02 | loss: 0.00 | backward: 0.03 | batch: 0.05
2021-02-21 16:35:59,676 (80/80) | batch: 31.28ms | Total: 0:00:03 | ETA: 0:00:01
2021-02-21 16:36:03,412 Epoch 9, MPJPE: 225.2374, PA-MPJPE: 138.5758, ACCEL: 3.2503, PVE: 262.0656, ACCEL_ERR: 9.4404,
2021-02-21 16:36:03,413 Epoch 10 performance: 138.5758
2021-02-21 16:36:05,412 Best performance achived, saving it!
2021-02-21 16:38:11,024 (1000/1000) | Total: 0:02:03 | ETA: 0:00:01 | loss: 4.9788 | loss_2d_kp: 0.35 | loss_3d_kp: 1.88 | loss_shape: 0.17 | loss_pose: 2.16 | data: 0.46 | forward: 0.02 | loss: 0.00 | backward: 0.03 | batch: 0.51
2021-02-21 16:38:14,089 (80/80) | batch: 29.62ms | Total: 0:00:02 | ETA: 0:00:01
2021-02-21 16:38:17,571 Epoch 10, MPJPE: 222.4956, PA-MPJPE: 133.9325, ACCEL: 3.3760, PVE: 258.5785, ACCEL_ERR: 9.5898,
2021-02-21 16:38:17,572 Epoch 11 performance: 133.9325
2021-02-21 16:38:19,488 Best performance achived, saving it!
2021-02-21 16:40:24,216 (1000/1000) | Total: 0:02:02 | ETA: 0:00:01 | loss: 4.9813 | loss_2d_kp: 0.38 | loss_3d_kp: 2.26 | loss_shape: 0.23 | loss_pose: 3.27 | data: 0.00 | forward: 0.02 | loss: 0.00 | backward: 0.03 | batch: 0.05
2021-02-21 16:40:27,282 (80/80) | batch: 30.03ms | Total: 0:00:03 | ETA: 0:00:01
2021-02-21 16:40:30,721 Epoch 11, MPJPE: 234.3234, PA-MPJPE: 134.5094, ACCEL: 3.8211, PVE: 271.2614, ACCEL_ERR: 9.9169,
2021-02-21 16:40:30,722 Epoch 12 performance: 134.5094
